views/dashboard.pug

```diff
@@ -1,8 +1,26 @@
-doctype html
-html(lang='en')
- head
-   title Dashboard
- body
-   h1 User
-   div.remark
-     p #{user.get("email")}
+include partials/header.pug
+section.jumbotron.d-flex.flex-row.justify-content-end
+    h1.mr-5.text-info 
+      | Dashboard 
+section.container 
+  div.remark
+    p User logged in with email "#{user.get("email")}".
+  form.column(method='post' action='/eventBuddyRequest')
+    .form-group.col-md-5
+      label.mr-sm-2(for='inlineFormCustomSelect') Preference
+      select.custom-select.mr-sm-2(id='event_type' name='event_type') Event type
```

Comment:
These are the event-types that were on the buddy sign up form jule shared friday evening: ["Theater_Kunst", "Ausflüge_Cafe", "Sport_Action", "Konzerte_Festivals", "Natur_Sightseeing", "Andere"]
This looks just as good, although splitting nature and sightseeing might make it harder to import the data from the 3700 volunteers, so I'd keep them together for now. And we should keep in mind that the labels are clear (e.g. sports is both doing and watching, café is also excursions, concert is also festival (not everyone will makethat link)).
And adding something for Theatre and art and for 'other' so the existing data can be applied upon.

Reply:
@julianvogels

Reply:
Fixed!